pkg/entities/nfts.go

```diff
@@ -52,12 +52,22 @@ func (e *Entity) GetNFTDetail(symbol, tokenID string) (*response.IndexerNFTToken
 	collection, err := e.repo.NFTCollection.GetBySymbol(symbol)
 	// cannot find collection in db
 	if err != nil {
+
 		if errors.Is(err, gorm.ErrRecordNotFound) {
-			err = fmt.Errorf("database: record nft collection not found")
+			// get collection
+			collection, err = e.repo.NFTCollection.GetByName(symbol)
```

Comment:
why need to do this ?
you just need to get collection first, then you have the symbol collection. Then no need to get by name